js/map.js

```diff
@@ -246,8 +246,9 @@ class CourseMap extends Backbone.Controller {
       let t = this.get('displayTitle');
       if (isStringEmpty(t)) t = this.get('title');
       if (isStringEmpty(t)) t = this.get('_id');
+      
       // Strip HTML tags
-      let title = document.createElement("div");
+      const title = document.createElement("div");
```

Comment:
I think jquery has a function for this:

`$(t).text()`

https://api.jquery.com/text/

Reply:
Thanks, that works. I still have to create the element so that jquery can retrieve the text value.

Reply:
Sorry, I forgot the wrapping